README.md

```diff
@@ -50,19 +50,19 @@ Dict{LightGraphs.SimpleGraphs.SimpleEdge{Int64},Tuple{Array{Float64,1},Array{Flo
 This plots are from the examples in
 [test/examples.jl](test/examples.jl), plotted via script in [test/demos.jl](test/demos.jl).
 
-
-![cross](./test/references/Zarate/cross.png "cross")
-
-![loop](./test/references/Zarate/loop.png "loop")
-
-![medium_pert](./test/references/Zarate/medium_pert.png "medium_pert")
-
-![sankey_3twos](./test/references/Zarate/sankey_3twos.png "sankey_3twos")
-
-![tiny_depgraph](./test/references/Zarate/tiny_depgraph.png "tiny_depgraph")
-
-![tree](./test/references/Zarate/tree.png "tree")
-
-![two_lines](./test/references/Zarate/two_lines.png "two_lines")
-
-![xcross](./test/references/Zarate/xcross.png "xcross")
+As you can see in the first plot below, pathing though the dummy nodes actually guarantees minimal number of crossings.
+On the _tiny_depgaph_ example direct has 3 crossings, where as path through dummy node only 2).
+However, it also doesn't actually always look as nice.
+It might look as nice if you used some swishy Bezier curves though.
```

Comment:
> It might look as nice if you used some swishy Bezier curves though.

I just found out that Plots exports `curves`, with it, the first example looks like this : 

![medium_pert_curved](https://user-images.githubusercontent.com/2031026/98481542-4f8bc500-21fb-11eb-9ba6-b249a8475950.png)

Reply:
Nice!

Reply:
I don't know, it surely looks better but it doesn't help explaining what is returned in `paths`

Reply:
What if we stick a scatter plot on top with big markers dummy nodes,
or insert the letter X in place of the node name for dummy nodes?

Reply:
Or we could leave as is, maybe adding mention of the curves function in the comment in the readme?

Reply:
```suggestion
It might nicer if you used some swishy Bezier curves though (Plots.jl has a [curves attribute](http://docs.juliaplots.org/latest/generated/graph_attributes/), and a matching function, which does this)
```